packager/media/base/test/rsa_test_data.cc

```diff
@@ -326,122 +326,125 @@ const char kTestMessage[] =
     "A fool thinks himself to be wise, but a"
     " wise man knows himself to be a fool.";
 
+// Self-generated test vector.  Used to verify algorithm stability.
 const uint8_t kTestEncryptedMessage_3072[] = {
```

Comment:
Can you keep the original encrypted message to verify it can still be decrypted successfully? The same for the signature verification. It can confirm "partially" that the algorithm is compatible with the original implementation.

Reply:
No, because I can't make the PRNG from mbedtls produce _exactly_ the same outputs as the one from OpenSSL.  Even with the same seed data, their outputs are different.

Reply:
Sorry for not making it clear. 

What I meant is to keep the original data but don't perform the data verification on that data.

Let's say the original gold encrypted data is called kOpensslGoldEncryptedData.

There can be two tests:

Test_1:
- Verify that kOpensslGoldEncryptedData can be decrypted and the decrypted message matches kTestMessage.

Test_2a:
- Verify that kTestMessage can be encrypted and then decrypted and the decrypted message matches kTestMessage.

Test_2b:
- Verify that the kTestMessage can be encrypted and the encrypted message matches the new gold encrypted data and then the encrypted message can be decrypted and the decrypted message matches kTestMessage.

Either test_2a or test_2b can be used.

Reply:
I see.  Yes, I could do that.  I'm out of time for today, but I'll make that change soon.

Reply:
Done!  This exposed my mistake in the signature code, and I fixed it by doing an explicit hash first.